README.md

```diff
@@ -112,6 +113,38 @@ instanceofDefinition.CONSTRUCTORS.MyClass = MyClass;
 ajv.validate({ instanceof: 'MyClass' }, new MyClass); // true
 ```
 
+### `instanceof`
+
+A specific case of `instanceof`. Returns true if passed an object or constructor of
+a class extending the passed schema parameter.
+
+The value of the keyword should be a string (`"Object"`, `"Array"`, `"Function"`, `"Number"`, `"String"`, `"Date"`, `"RegExp"`, `"Promise"` or `"Buffer"`) or array of strings.
+
+To pass validation the result of `data subclassof ...` operation on the value should be true:
+
+```javascript
+class ChildArray extends Array {}
+class ChildFunction extends Function {}
+ajv.validate({ subclassof: 'Array' }, ChildArray); // true
+ajv.validate({ subclassof: 'Array' }, new ChildArray); // true
+ajv.validate({ subclassof: 'Array' }, []); // false
+ajv.validate({ subclassof: ['Array', 'Function'] }, new ChildArray); // true
+ajv.validate({ subclassof: ['Array', 'Function'] }, new ChildFunction); // true
+```
+
+You can add your own constructor function to be recognized by this keyword:
+
+```javascript
+class MyClass {}
+class MyChildClass extends MyClass {}
+var subclassofDefiniton = require('ajv-keywords').get('subclassof');
+// or require('ajv-keywords/keywords/subclassof');
+var ajv = defFunc(new Ajv(), { MyClass: MyClass });
```

Comment:
It seems incorrect

Reply:
Woops. Changing to `subclassofDefiniton`